webapp/views.py

```diff
@@ -32,7 +29,7 @@
 #needed for user control and authentication
 from django.contrib.auth import authenticate, login, logout
 # Needed for permission control
-from django.contrib.auth.decorators import permission_required, login_required

```

Comment:
just got rid of some unused imports in this file. my editor fixed the windows line breaks as well